docs-cg.md

```diff
@@ -0,0 +1,67 @@
+# W3C Documentation CG Charter
+
+## Mission
+
+The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform. The Docs CG focuses on the art of technical writing and information architecture across the Web ecosystem (W3C, WHATWG, ECMA, IETF, etc.).
```

Comment:
I think coordinating horizontal efforts is just as important for this group than focusing on the art of writing and IA. So I'd actually add this to the mission statement as well.

If I'm a web dev vaguely interested in the CG, I fear I'd be a bit intimidated by the mission and might not continue reading. I'd think: I'm not a writer, this isn't for me.

So I think this intro/mission could do a bit of a better job representing more of the scope of the work, and appealing to all of the target audience.

In fact, maybe the mission can be shorter:

```suggestion
The mission of the W3C Documentation Community Group (W3C Docs CG) is to ensure web developers and designers worldwide have the best information available so they can build things on the web platform.
```